applications/asset_tracker_v2/src/modules/cloud_module.c

```diff
@@ -224,6 +225,16 @@ static bool app_event_handler(const struct app_event_header *aeh)
 
 		msg.module.cloud = *evt;
 		enqueue_msg = true;
+
+		/* If the event is an event that only the cloud module cares about,
```

Comment:
Maybe write something like "events that are intended to only be used by the cloud module"